sig-apps/CONTRIBUTING.md

```diff
@@ -1,64 +1,35 @@
+and [Contributor Cheat Sheet](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet) first.
 
-To get an idea of what's going on, here's an overview of each of the areas.
+## Getting Started
 
-### Workloads API
+* If you're a newcomer and have no idea where to start, we have a non-stale pool of issues that are
+available for you:
+  * [first-good-issue](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+label%3Asig%2Fapps+)
+  * [help wanted](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+label%3Asig%2Fapps+)
 
-The _Workloads API_ including controllers and jobs is part of SIG Apps.The
+* The _Workloads API_ including controllers and jobs is part of SIG Apps.The
 controllers are Deployments, DaemonSets, StatefulSets, and ReplicaSets. The jobs
 are part of the batch API and include Jobs and CronJobs. These are part of core
-Kubernetes and developed within the [monorepo](https://github.com/kubernetes/kubernetes).
-
-Don't let the fact that these are part of core Kubernetes intimidate you. Each
-controller is its own package that can be found in the [pkg/controllers directory](https://github.com/kubernetes/kubernetes/tree/master/pkg/controller). Each
-controller is small.
-
-Like other parts of core Kubernetes, you can learn about contributing code or
-reviewing pull requests via the [Contributors Guide](https://github.com/kubernetes/community/tree/master/contributors/guide).
-
-### Application CRD and Controller
-
-The [Application CRD and controller](https://github.com/kubernetes-sigs/application)
-provides a method to group application resources and describe the application.
+Kubernetes and developed within the [monorepo](https://github.com/kubernetes/kubernetes).Each controller has its own package that can be found in the [pkg/controllers directory](https://github.com/kubernetes/kubernetes/tree/master/pkg/controller).
+* The [Application CRD and controller](https://github.com/kubernetes-sigs/application) provides a method to group application resources and describe the application.
 This SIG sponsored project has a fairly minimal and straight forward [contributing
 process](https://github.com/kubernetes-sigs/application/blob/master/CONTRIBUTING.md)
 
-### Kompose
-
-[Kompose](https://github.com/kubernetes/kompose) stands for Kubernetes + Compose
+* [Kompose](https://github.com/kubernetes/kompose) stands for Kubernetes + Compose
 (the docker tooling). This project translates Docker compose configuration into
-Kubernetes configuration.
-
-Kompose has its own [contributing guide](https://github.com/kubernetes/kompose/blob/master/CONTRIBUTING.md).
-
-## Contributing Beyond Code
-
-There are a couple ways to contribute beyond code and the sub-projects. They include:
+Kubernetes configuration. Kompose has its own [contributing guide](https://github.com/kubernetes/kompose/blob/master/CONTRIBUTING.md).
 
```

Comment:
probably could be good to add that we are seeking user use cases and feedback on new features and that it would help us if users would give their opinions on new KEPs and try alpha features